src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        int index = random.nextInt(Colors.values().length);
+        Colors colors = Colors.values()[index];
+        return String.valueOf(colors);
```

Comment:
You get one color instead colors and use color.name, you don`t need String.valueOf